### DISCUSSION TITLE
Anyone testing Netbox with Ubuntu 22.04 yet?

### DISCUSSION THREAD

**Discussion by TheDirkMelvin** [netbox-community/netbox#9117] [General]
I have not attempted to try Netbox on anything other than Ubuntu 20.04, since I had a very rough start with Netbox. Just curious if anyone here has been using/testing Netbox on newer editions of Ubuntu.

**Comment by joaolucasmacedo** [comment]
I think that would be no problem. You better just check your Python version.

**Reply by dreng** [reply]
As long as your packages are up to date python shouldn't be a problem either. The most recent version für Ubuntu 20.4 is python 3.8. See https://packages.ubuntu.com/focal/python/python3

If you are still uncertain make a backup / snapshot or use a test system like a VM. Personally I always take a snapshot of our production VM before updating.

**Comment by mskalecki** [comment]
After doing an in-place upgrade from 20.04 to 22.04, the netbox service was unable to start, because the VENV needed to be rebuilt.

The simple fix was to run the upgrade.sh script in the netbox installation directory.

**Comment by ZPrimed** [comment]
I wouldn't be too worried about NetBox working correctly on 22.04; I *would* be a little worried about NAPALM (if you use it) working correctly, though. It is fairly picky about versioning of components, especially if you are using 3rd party equipment "drivers" (which if you're using more than Cisco & JunOS, good chance you are using a 3rd party driver).
